### DISCUSSION TITLE
PathTooLongException: Is there a way to shorten file names?

### DISCUSSION THREAD

**Discussion by cumiller** [Doraku/DefaultDocumentation#63] [Q&A] [answered]
I am getting an error indicating the the "specified path, file name, or both are too long" - System.IO.PathTooLongException.

Here's an example file (names obscured):
C:\Docs\SSSSS_Iiiiiiiiiii_SSSSSAaaaaaa_BbbbbbbbLlllll_RrrrrrrrrrrBbbbbbbbLllll_UuuuuuBbbIiii(SSSSS_Iiiiiiiiiii_Bbbbbbb_Ssssss_Collections_Generic_List_SSSSS_Iiiiiiiiiii_dbBbbbbbbAllowance__Ssssss_Collections_Generic_List_SSSSS_Iiiiiiiiiii_dbBbbbbbbItem__bool).md

Is there a option to produce short names?

**Comment by Doraku** [comment, marked as answer]
There is, check [`DefaultDocumentationFileNameMode`](https://github.com/Doraku/DefaultDocumentation#msbuild-task) in the readme.

**Reply by cumiller** [reply]
Perfect. Thanks!

**Reply by jason-c-daniels** [reply]
I've just browsed to the link provided and cannot find the text "DefaultDocumentationFileNameMode" anywhere in the linked page. Has it been relocated elsewhere?

**Reply by Doraku** [reply]
Yeah sorry this settings was changed to [FileNameFactory](https://github.com/Doraku/DefaultDocumentation#DocItemConfiguration_FileNameFactory) in the latest version

**Reply by jason-c-daniels** [reply]
Awesome thanks!
